src/Categories/Monad/Construction/Kleisli.agda

```diff
@@ -0,0 +1,116 @@
+module Categories.Monad.Construction.Kleisli where
+
+-- Definition of kleisli triple as a relative monad
+-- and the equivalence of kleisli triple and monad
+-- see https://ncatlab.org/nlab/show/extension+system (the naming differs)
+
+open import Level
+open import Categories.Category
+open import Categories.Monad.Relative renaming (Monad to RMonad)
+open import Categories.Monad
+open import Categories.Functor renaming (id to idF)
+open import Categories.NaturalTransformation renaming (id to idN)
+import Categories.Morphism.Reasoning as MR
+
+private
+    variable
+        o ℓ e : Level
+
+module _ (C : Category o ℓ e) where
+    open Category C
+    open HomReasoning
+    open Equiv
+    open MR C
+
+    -- a kleisli triple is a relative monad with J = idF
+    Kleisli : Set (o ⊔ ℓ ⊔ e)
```

Comment:
I'm wondering if the type should be called `KleisliTriple` as there are too many `Kleisli` things already? (The implication below can keep its name)

Reply:
As a slightly polemic point, nLab seems to promote the name "extension system" https://ncatlab.org/nlab/show/extension+system. I am more used to the name "Kleisli triple", but nLab may well change the trend. I do not really have any opinion on this.

Reply:
I would be fine with either `KleisliTriple` or `ExtensionSystem`.

Reply:
I chose `KleisliTriple` and `KleisliCotriple` for now.